project-base/src/Shopsys/ShopBundle/Resources/styles/front/common/components/list/categories.less

```diff
@@ -40,40 +40,22 @@
 
             text-decoration: none;
             border: 1px solid @color-border;
-
-            .is-no-flex & {
-                display: block;
-
-                font-size: 0;
-            }
         }
 
         &__image {
             padding: @list-categories-item-padding 0 @list-categories-item-padding @list-categories-item-padding;
-            flex: 0 @list-categories-item-image-width;
+            width: @list-categories-item-image-width;
             text-align: center;
-
-            .is-no-flex & {
-                display: inline-block;
-                vertical-align: middle;
-                width: @list-categories-item-image-width;
-            }
         }
 
         &__title {
-            flex: 1;
+            width: calc(~"100% - @{list-categories-item-image-width}");
```

Comment:
Please check if you can substitute `flex` property for the `width` property.

Reply:
Flex: 1 sets all flexible items to be same lenght. Is not necessary bcs, we set calculated width

Reply:
It is okay for now. We will refactor this later.